src/event-sources/utils.js

```diff
@@ -148,6 +152,17 @@ function getCommaDelimitedHeaders ({ headersMap, separator = ',', lowerCaseKey =
 
 const emptyResponseMapper = () => {}
 
+function emptyResponseMapperToEventSource({
+    statusCode,
+    body,
+}) {
+  if (statusCode >= 400) {
+    throw new Error(body)
```

Comment:
Could you explain this? Seems like it might be a breaking change

Reply:
It is same as the above
https://github.com/CodeGenieApp/serverless-express/pull/668#discussion_r1433596225

Reply:
Let's revert this also.

Reply:
I reverted